docs/node_disruption.md

```diff
@@ -5,4 +5,13 @@ The `nodeFailure` field triggers a kernel panic on the node. Because the node wi
 * `/proc/sys/kernel/sysrq` > `/mnt/sysrq`
 * `/proc/sysrq-trigger` > `/mnt/sysrq-trigger`
 
-> :warning:️ Node behavior when using this disruption can differ depending on the cloud provider (node may or may not be replaced, restarted, cordoned, etc.).
\ No newline at end of file
+> :warning:️ Node behavior when using this disruption can differ depending on the cloud provider (node may or may not be replaced, restarted, cordoned, etc.).
+
+## Targeting
+
+For clarity purpose, it is mandatory to explicitly set the disruption's `level` field to either `pod` or `node`.
+
+The `nodeFailure` disruption acts on a node. However, the `level` field needs to be adapted for the selector:
+
+- if `level: node` is set, the selector will target nodes and impact those nodes directly.
```

Comment:
Hence level is somehow misleading and should be part of selector potentially.

If we revamp the spec we should probably have something like:

```yaml
  selector:
    labels:
      test: value
    annotations:
      test: value
    level: node|pod
```